timesketch/lib/analyzers/phishy_domains.py

```diff
@@ -40,14 +40,15 @@ def __init__(self, index_name, sketch_id):
             sketch_id: Sketch ID
         """
         self.index_name = index_name
-        super(DomainsSketchPlugin, self).__init__(index_name, sketch_id)
+        super(PhishyDomainsSketchPlugin, self).__init__(index_name, sketch_id)
 
         self.domain_scoring_threshold = current_app.config.get(
             'DOMAIN_ANALYZER_WATCHED_DOMAINS_SCORE_THRESHOLD', 0.75)
         self.domain_scoring_whitelist = current_app.config.get(
             'DOMAIN_ANALYZER_WHITELISTED_DOMAINS', [])
 
-    def _get_minhash_from_domain(self, domain):
+    @staticmethod
```

Comment:
why are you making this a staticmethod???

Reply:
and not just have a self, domain there like it was.... don't see the need for that

Reply:
heh, as discussed offline. Making it static is best practise when you don't need a reference to the object it self. That will reduce the amount of objects created. but this is moot now, because these methods are now refactored to utils.py instead.